src/app/trips/trip-forms/trip-form/trip-form.component.html

```diff
@@ -0,0 +1,165 @@
+        [isRequired]="true">
+      </form-field>
+    </div>
+
+    <!-- title VALIDATION ALERTS -->
+    <div class="col-xs-12">
+      <div *ngIf="this.title?.touched && this.title?.invalid">
+        <!-- required -->
+        <validation-alert
+          [validationBoolean]="!!this.title?.errors && !!this.title?.errors!['required']"
+          [alertText]="'Title is required.'">
+        </validation-alert>
+
+        <!-- min-length -->
+        <validation-alert
+          [validationBoolean]="!!this.title?.errors && !!this.title?.errors!['minlength']"
+          [alertText]="'Title must have at least 3 characters.'">
+        </validation-alert>
+      </div>
+    </div>
+  </div>
+  <!--  -->
+
+
+  <!-- DATES (required) -->
+  <div>
+    <div formGroupName="datesGroup">
+      <!-- START DATE -->
+        <div class="col-xs-6 control">
+          <!-- label & input component -->
+          <form-field
+            [$formControlName]="'startDate'"
+            [labelText]="'Start Date'"
+            [$type]="'date'"
+            [hasInvalidStyling]="startDate?.touched && startDate?.invalid"
+            [isRequired]="true">
+          </form-field>
+        </div>
+        <!-- END DATE -->
+        <div class="col-xs-6 control">
+          <!-- label & input component -->
+          <form-field
+            [$formControlName]="'endDate'"
+            [labelText]="'End Date'"
+            [$type]="'date'"
+            [hasInvalidStyling]="endDate?.touched && endDate?.invalid"
+            [isRequired]="true">
+          </form-field>
+        </div>
+      </div>
+      <!-- start date & end date VALIDATION ALERTS -->
+      <div class="col-xs-12">
+        <!-- START DATE validation alert -->
+        <div class="alert-lefthalf">
+          <!-- must match pattern and fit in valid time frame -->
+          <validation-alert
+            [validationBoolean]="startDate?.touched && startDate?.invalid"
+            [alertText]="'Date must be valid. Valid time frame: 01/01/1900 - 01/01/2100'">
+          </validation-alert>
+        </div>
+        <!-- END DATE validation alert -->
+        <div class="alert-righthalf">
+          <!-- must match pattern and fit in valid time frame -->
+          <validation-alert
+            [validationBoolean]="endDate?.touched && endDate?.invalid"
+            [alertText]="'Date must be valid. Valid time frame: 01/01/1900 - 01/01/2100'">
+          </validation-alert>
+        </div>
+        <!-- DATES NOT IN ORDER validation alert (startDate > endDate) -->
+        <validation-alert
+          [validationBoolean]="!!this.datesGroup?.errors"
+          [alertText]="'End Date must occur after Start Date.'">
+        </validation-alert>
+      </div>
+  </div>
+  <!--  -->
+
+
+  <!-- DESTINATIONS (required) -->
+  <div>
+    <!-- Dynamic Input -->
+    <form-array
+      [$formArrayName]="'destinations'"
+      [$placeholder]="'City,  Region,  Country'"
```

Comment:
putting [] around an attribute means you want to write a code expression to resolve it. When you just have a string to provide it becomes confusing because you need to provide single quotes in the double quotes. You should take the square brackets out and just provide the string you want to use when you don't really need to provide an expression.